File: Angular/Restaurant Part/src/app/services/menu-item.ts
export interface MenuItem {
  menuItemId?: number;         // Optional: for new items before save
  name: string;                // Name of the menu item
  category: string;            // Category like "Burger", "Pizza"
  price: number;               // Price in ₹
  dietaryInfo: string;         // Veg / Non-Veg
  description: string;         // Full description
  tasteInfo?: string;          // Optional: Spicy, Sweet, etc.
  nutritionalInfo?: string;    // Optional: "500 kcal"
  imageUrl?: string;           // Optional: link to image
  isAvailable: boolean;        // Is the item currently available?
  restaurantId: number;        // Foreign key to restaurant
}

File: Web API and MVC/Models/MenuItem.cs
namespace HotBytee.MVC.Models
{
    public class MenuItem
    {
        public int MenuItemId { get; set; }  // Primary Key
        public string Name { get; set; }
        public string Description { get; set; }
        public string Category { get; set; }  // Breakfast, Lunch, etc.
        public decimal Price { get; set; }
        public string ImageUrl { get; set; }
        public bool IsAvailable { get; set; }
        public string TasteInfo { get; set; }  // Spicy, Sweet, etc.
        public string NutritionalInfo { get; set; }  // Optional
        public string DietaryInfo { get; set; } // Veg / Non-Veg

        public int RestaurantId { get; set; }
    }
}